backend/src/packages/auth/libs/helpers/assert-jwt-payload.helper.ts

```diff
@@ -0,0 +1,12 @@
+import { type JwtPayload } from 'shared/build/index.js';
+
+const assertJwtPayload = (payload: unknown): payload is JwtPayload => {
+  return (
+    typeof payload === 'object' &&
+    !!payload &&
+    'id' in payload &&
+    typeof payload.id === 'number'
```

Comment:
So why not joi?

Reply:
because using joi i would eventually use "as". But if you have nothing against it, I will redo

Reply:
Forget it, I've added Joi